### PR TITLE
Support training with custom model image

### PR DESCRIPTION
**Pull request by typhoonzero**

### CODE REVIEW

File: pkg/sql/ir_generator.go [hunk 1]
```diff
@@ -81,6 +81,7 @@ func generateTrainStmt(slct *parser.SQLFlowSelectStmt, attrInitAndTypeCheck bool
 	modelParts := strings.Split(modelURI, "/")
 	modelImageName := strings.Join(modelParts[0:len(modelParts)-1], "/")
 	modelName := modelParts[len(modelParts)-1]
+	fmt.Printf("image: %s, modelname: %s", modelImageName, modelName)
```
Comment (Yancey0623) [inline pkg/sql/ir_generator.go]: better to remove `fmt.Printf`, if it's necessary, please use `log.Printf`.

**Review by Yancey0623 — COMMENTED** [review]
Do you want to perfect the CI with the custom model image?

**Review by Yancey0623 — APPROVED** [review]
LGTM

File: cmd/sqlflowserver/main_test.go [hunk 2]
```diff
@@ -2000,6 +2002,42 @@ FROM %s LIMIT 5;
...
+SELECT * FROM %s LIMIT 10;
+
+SELECT * FROM %s
+TO TRAIN %s/DNNClassifier
+WITH
+	model.n_classes = 3,
+	model.hidden_units = [64, 32],
+	validation.select = "SELECT * FROM %s"
+LABEL class
+INTO test_workflow_model;`, caseTrainTable, caseTrainTable, customImage, caseTestTable)
```
Comment (tonyyang-svail) [inline cmd/sqlflowserver/main_test.go:2022]: Do we need to add tests on PREDICT/EXPLAIN?

Comment (typhoonzero) [reply cmd/sqlflowserver/main_test.go:2022]: Unfortunately predict and explain are not supported until we can find a way to analyze the SQL program and find out the image used by train and apply it to the predict SQL statement.

**Pull request merged**